dts/examples/nucleo_u5a5_autotest.dts

```diff
@@ -31,8 +31,8 @@
 
 /{
 	reserved-memory {
-		autotest_code: autotest_code@800a000 {
-			reg = <0x800a000 0xe000>;
+		autotest_code: autotest_code@800d000 {
```

Comment:
Ah, it will probably break the projects build requiring `reserved-memory` sections reconfiguration ! 
Could it be done as KConfig-based from the beginning with DISABLED as default value ?

Reply:
added in #89, that needs to be merged after that one.